src/app/pages/vm/devices/device-form/device-form.component.ts

```diff
@@ -303,6 +306,25 @@ export class DeviceFormComponent implements OnInit {
 
   onSubmit(event: SubmitEvent): void {
     event.preventDefault();
+
+    if (this.typeControl.value === VmDeviceType.Pci) {
+      this.ws.call('vm.device.passthrough_device_choices')
+        .pipe(untilDestroyed(this)).subscribe((passthroughDevices) => {
+          if (!passthroughDevices[this.pciForm.controls.pptdev.value]?.reset_mechanism_defined) {
```

Comment:
Shouldn't this also check if the device in question is an isolated GPU? The ticket states you can check using `isolated_gpu_pci_ids` field returned by `system.advanced.config"`